fontforgeexe/fontview.c

```diff
@@ -1016,13 +1016,13 @@ void MenuExit(GWindow UNUSED(base), struct gmenuitem *UNUSED(mi), GEvent *e) {
 	DelayEvent(_MenuExit,NULL);
 }
 
-char *GetPostScriptFontName(char *dir, int mult) {
+char *GetPostScriptFontName(char *dir, bool mult, bool modal) {
```

Comment:
If you have an inkling to be truly heroic, you could:

1. Move `GetPostScriptFontName()` from this file to `openfontdlg.c`.
2. Remove the `FVOpenFont()` entry from `views.h`
3. Make `FVOpenFont()` static. 

This is how the code should be arranged given that `GetPostScriptFontName()` is basically a wrapper and the only place `FVOpenFont()` is called.

Reply:
Thanks. I do rather like being heroic, (don't we all?), and if I don't do it it might not get done for another ten years, so I did it. Once CI passes I'll merge